todoman/cli.py

```diff
@@ -466,6 +468,8 @@ def move(ctx, list, ids):
 @cli.command()
 @pass_ctx
 @click.argument('lists', nargs=-1, callback=_validate_lists_param)
+@click.option('--description',
```

Comment:
Since you deleted the implementation for this flag, the flag should be removed too.

Note that that implementation was not wrong; just missing tests though.